api/v3/VolunteerUtil.php

```diff
@@ -223,13 +203,20 @@ function _volunteerGetProjectRelationshipDefaults() {
       'relationship_type_id' => $employerRelationshipTypeId,
       'is_active' => 1,
     ));
-    $defaultBeneficiary = array($result);
+    $defaultBeneficiary = array('contact_id' => $result);
   } catch(Exception $e) {
     $domain = civicrm_api3('Domain', 'getsingle', array('current_domain' => 1));
-    $defaultBeneficiary = array($domain['contact_id']);
+    $defaultBeneficiary = array('contact_id' => $domain['contact_id']);
   }
   $defaults[$beneficiaryType] = $defaultBeneficiary;
 
+  foreach ($defaults as $type => $default) {
+    $defaults[$type]['can_be_read_by_current_user'] = CRM_Volunteer_BAO_ProjectContact::contactIsReadable($default['contact_id']);
```

Comment:
Yeah, my hack in line 217 matches the format but doesn't offer the benefits of the format. Happy for you to fix this or to leave it for later, since it has no ramifications at the moment.